about us/about_us.js

```diff
@@ -0,0 +1,341 @@
+    checkSign.checked = true;
+    checkSign.classList.add("checkSign");
+
+    candidateTraitDescription.append(candidateTrait);
+    candidateTraitBox.append(checkSign);
+    candidateTraitBox.append(candidateTraitDescription);
+    candidateTraitsBox.append(candidateTraitBox);
+  }
+}
+
+const programStages = [
+  {
+    programStage: "Pre-selection",
+    programStageInfo:
+      "Candidate completes a self-assessment questionnaire. Initial stage will consist of the candidate taking intro classes of Javascript, math (if needed), and english (if needed). The candidate will have a mentor who works with him/her once a week. The intro js course is online and it’s free. The candidate will take this mostly on his/her own time.",
+  },
+  {
+    programStage: "Bootcamp",
+    programStageInfo:
+      "If/when the candidate successfully completes the pre-selection course, he/she will enroll into a bootcamp which the program will pay for. At this stage we will also allocate more time with the mentor. The bootcamp will be either another intro or advanced course, which will depend on the level of knowledge with which the candidate graduates from the pre-selection phase.",
+  },
+  {
+    programStage: "Practice",
+    programStageInfo:
+      "If the candidate has shown real promise and dedication, he/she will enter the practice stage. To become a hirable engineer, usually mid level, a great deal of practice is needed. The mentor will give more tutorials and projects to the candidate to prepare him/her for production.",
+  },
+  {
+    programStage: "Internship",
+    programStageInfo:
+      "The candidate will join a real project and work in a real production environment.",
+  },
+  {
+    programStage: "Job hunt",
+    programStageInfo:
+      "The candidate starts interviewing for positions. We will help with writing his/her resume and mock interviews. We can also use our contacts to help land some interviews.",
+  },
+];
+
+const programStagesBox = document.getElementById("program-stages");
+const programStageInfoElem = document.getElementById("program-stage-info");
+let index = 1;
+function renderProgramStageInfo(programStages) {
+  for (let programStage of programStages) {
+    const programStageBox = document.createElement("div");
+    programStageBox.classList.add("programStageBox");
+    programStageBox.id = programStage.programStage;
+
+    const programStageNameElem = document.createElement("p");
+    programStageNameElem.append(programStage.programStage);
+    const programStageInfo = document.createElement("p");
+    programStageInfo.append(programStage.programStageInfo);
+    programStageBox.append(programStageNameElem);
+    programStagesBox.append(programStageBox);
+
+    // append arrow
+    if (index !== programStages.length) {
+      const arrow = document.createElement("span");
+      arrow.innerHTML = "&rarr;";
+      programStagesBox.append(arrow);
+      arrow.classList.add("arrow");
```

Comment:
see how you put the comment above to describe what this small block is doing? This is a good opportunity to move this code into a function called appendArrow instead of using the comment